src/main/java/hudson/plugins/parameterizedtrigger/BuildInfoExporterAction.java

```diff
@@ -129,6 +156,111 @@ public BuildReference(final String projectName) {
       this.buildNumber = 0;
       this.buildResult = Result.NOT_BUILT;
     }
+
+    public String getProjectName() {
+      return projectName;
+    }
+
+    public int getBuildNumber() {
+      return buildNumber;
+    }
+
+    public Result getBuildResult() {
+      return buildResult;
+    }
+
+    @Override
+    public void update() {
+
+    }
+  }
+
+  public static class StaticBuildReference extends AbstractBuildReference {
```

Comment:
@kwhetstone 
Hi, thanks for the review.
I added a super class to the build reference and added two subclasses, one to get the information from the currently running build which needs to queried from Jenkins on every access (didn't found a way to do it different). The other one the StaticBuildReference is used when Jenkins serializes data between shutdown and restart which should do the same as BuildReference. However I kept the BuildReference since I didn't want to break the deserialization. It's a long time since I have used the built-in java serialization in combination with class renaming and I as far as I know it's impossible using a different class without being able to reimplement readClassDescriptor on the ObjectInputStream. However the ObjectInputStream is created by jenkins. Do you suggest just to keep the old name?